### PR TITLE
Update map tip and legend on landing page for 0.5.1 release

### PR DESCRIPTION
**Pull request by pengyin-shan**
PENDING for docker build. Will change to formal PR after finishing the docker deployment.

Including all features that the client requested for the 0.5.1 release:

1. Change the tip title to 'Benefits' and 'Costs' for crop insurance and snap on the landing page. The SNAP page's map got updated accordingly.
2. Change the negative/zero number format for both legend and tips for the landing page maps. The SNAP page's map got updated accordingly.

### CODE REVIEW

**Comment by navarroc** [comment]
I know this is just a draft, but I just wanted to remind about changing package.json to 0.5.1

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
```
Comment (navarroc) [inline CHANGELOG.md:7]: This should also be 0.5.1 when ready

**Comment by pengyin-shan** [comment]
Ah

> I know this is just a draft, but I just wanted to remind about changing package.json to 0.5.1

Oh for sure, I planned to do this when creating pr from the release branch to main. But I can also do it now

**Comment by navarroc** [comment]
> Ah
> 
> > I know this is just a draft, but I just wanted to remind about changing package.json to 0.5.1
> 
> Oh for sure, I planned to do this when creating pr from the release branch to main. But I can also do it now
Ok, on release branch works too.

File: CHANGELOG.md (hunk 1 above)
Comment (pengyin-shan) [reply CHANGELOG.md]: updated

**Comment by navarroc** [comment]
@pengyin-shan Indiana, Illinois and Iowa have negative benefits, but the total is showing as positive now in the hover. IL has -535M, but it shows at $535M

**Comment by navarroc** [comment]
All others look fine from what I can tell, just IN, IA and IL now show a positive total benefit when it should be negative overall. The individual years show correctly

**Comment by pengyin-shan** [comment]
> All others look fine from what I can tell, just IN, IA and IL now show a positive total benefit when it should be negative overall. The individual years show correctly

thank you! let me check

**Comment by pengyin-shan** [comment]
> All others look fine from what I can tell, just IN, IA and IL now show a positive total benefit when it should be negative overall. The individual years show correctly

thank you for catching this! it should work now

**Comment by pengyin-shan** [comment]
> > All others look fine from what I can tell, just IN, IA and IL now show a positive total benefit when it should be negative overall. The individual years show correctly
> 
> thank you for catching this! it should work now

sorry one second, the 0 is not working correctly now. I'm fixing it

**Comment by pengyin-shan** [comment]
@navarroc very sorry for the wait...It should be ready now. I also squashed some previous commits.

**Review by navarroc — APPROVED** [review]

**Comment by navarroc** [comment]
Looks good from what I can see. I compared with dev and I don't see any issues.

File: CHANGELOG.md [hunk 2]
```diff
@@ -90,6 +95,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Map data json [#12](https://github.com/policy-design-lab/pdl-frontend/issues/12)
 - Final landing page changes for initial milestone [#15](https://github.com/policy-design-lab/pdl-frontend/issues/15)
 
+<<<<<<< HEAD
```
Comment (navarroc) [inline CHANGELOG.md:98]: Can you remove this?

Comment (pengyin-shan) [reply CHANGELOG.md]: oh yes, sorry somehow I missed the warning of this file

**Comment by pengyin-shan** [comment]
> 



> Looks good from what I can see. I compared with dev and I don't see any issues.

great, thank you!!

**Comment by navarroc** [comment]
This is deployed on dev.

**Comment by pengyin-shan** [comment]
> This is deployed on dev.

thank you Chris! I checked dev and it worked as expected

**Comment by navarroc** [comment]
Given we are short staff this week, you may want to merge this since you can override the 2 approvals needed and create a release branch so I can review while I'm available. Then @ywkim312 can do a final review Tuesday so we can release as long as it looks good to him too and we find no issues.

**Comment by pengyin-shan** [comment]
> Given we are short staff this week, you may want to merge this since you can override the 2 approvals needed and create a release branch so I can review while I'm available. Then @ywkim312 can do a final review Tuesday so we can release as long as it looks good to him too and we find no issues.

sounds good, I'll merge then

**Pull request merged**